iib/workers/tasks/build.py

```diff
@@ -784,6 +883,23 @@ def handle_add_request(
     _update_index_image_build_state(request_id, prebuild_info)
 
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
+        if from_index:
+            log.info('Checking if bundles are already present in index image')
```

Comment:
If the message is the same can we put it to variable?

Reply:
Done.